static/src/js/components/TextWindowActions/TextWindowActions.js

```diff
@@ -306,7 +306,7 @@ export const TextWindowActions = ({
               </a>
               or
               {' '}
-              <a className="link link--external" href="/earthdata-download">learn more.</a>
+              <a className="link link--external" href="https://nasa.github.io/earthdata-download/">learn more.</a>
```

Comment:
this has the `link--external` class, but it isn't opening in a new tab

Reply:
:doh: